src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleNull.cs

```diff
@@ -28,8 +28,6 @@ private static bool HandleNull(JsonSerializerOptions options, ref Utf8JsonReader
                 return true;
             }
 
-            Debug.Assert(jsonPropertyInfo != null);
```

Comment:
Why remove this?

Reply:
not needed, above if `jsonPropertyInfo == null` it returns